_modules/sesceph/__init__.py

```diff
@@ -161,14 +162,22 @@ def _update_partition(action, dev, description):
 
 
 
-def zap(dev):
+def zap(dev = None, **kwargs):
     """
     Destroy the partition table and content of a given disk.
     """
+    if dev is not None:
+        log.warning("Depricated use of function function use kwargs")
```

Comment:
spelling on Deprecated.  I didn't know why the word function is there
twice.  I'm thinking the message is "Deprecated use of function, use
kwargs".

On Mon, Apr 4, 2016 at 4:26 AM, Owen Synge <notifications@github.com> wrote:

> In _modules/sesceph/__init__.py
> <https://github.com/oms4suse/sesceph/pull/22#discussion_r58342010>:
>
> >      """
> >      Destroy the partition table and content of a given disk.
> >      """
> > +    if dev is not None:
> > +        log.warning("Depricated use of function function use kwargs")
>
> sorry I do not understand what you mean here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/oms4suse/sesceph/pull/22/files/84a48a3b3cd0c85bb689caab139fe6425fd7b5a8#r58342010>
>